src/miniscript/types/extra_props.rs

```diff
@@ -458,6 +531,13 @@ impl Property for ExtData {
                     .and_then(|x| l.ops_count_nsat.map(|y| y + x + 2)),
             ),
             ops_count_nsat: None,
+            stack_elem_count_sat: cmp::max(
+                l.stack_elem_count_sat,
+                l.stack_elem_count_dissat
+                    .and_then(|l_dis| r.stack_elem_count_sat.map(|r_sat| r_sat + l_dis)),
+            ),
+            // FIXME: why??
+            stack_elem_count_dissat: None,
```

Comment:
nit: FIXME can be removed. The typing rules ensure that output of or_c is V type. (cannot be dissatisfied)